Makefile.am

```diff
@@ -365,7 +373,8 @@ cpp_server_ct_dns_server_LDADD = \
 	cpp/libcore.a \
 	$(evhtp_LIBS) \
 	${libevent_LIBS} \
-	-lprotobuf -lldns -lsqlite3
+	$(sqlite3_LIBS) \
```

Comment:
This one seems spurious? Or the other below?

Reply:
Fixed.